### DISCUSSION TITLE
Include oxc_language_server binary in oxlint npm package

### DISCUSSION THREAD

**Discussion by alisnic** [oxc-project/oxc#3272] [Feature Request]
Currently, non-VSCode users have to build vscode package by hand and extract the binary if they want to use oxc language server as a vanilla language server

Bundling it in the npm package would make integration with other editors a lot easier

Related to https://github.com/oxc-project/oxc/pull/1836

**Comment by Boshen** [comment]
<img width="547" alt="image" src="https://github.com/user-attachments/assets/7685edb3-d9cf-4704-9cf8-f305b35b60f9">
